polaris-tokens/scripts/toJSON.js

```diff
@@ -1,22 +1,26 @@
 const fs = require('fs');
 const path = require('path');
 
-const {tokens} = require('../dist/index');
-
 function getFileName(fileName) {
   return path.join(__dirname, '../dist', fileName);
 }
 
-Object.entries(tokens).forEach(([tokenGroup, tokenProps]) => {
-  if (tokenGroup === 'colorSchemes') {
-    Object.entries(tokenProps).forEach(([colorTokenGroup, colorTokenProps]) => {
-      const fileName = getFileName(`colors.${colorTokenGroup}.json`);
+function toJSON(tokens) {
```

Comment:
Awesome! However, it looks like we're `promise.all`ing a synchronous util. I suggest updating to use `for..of` loops and `fs.promises.writeFile`.
```js
async function toJSON(tokens) {
  for (const [tokenGroup, tokenProps] of Object.entries(tokens)) {
    if (tokenGroup === 'colorSchemes') {
      for (const [colorTokenGroup, colorTokenProps] of Object.entries(tokenProps)) {
          const fileName = getFileName(`colors.${colorTokenGroup}.json`);

          await fs.promises.writeFile(fileName, JSON.stringify(colorTokenProps));
        },
      );
    } else {
      const fileName = getFileName(`${tokenGroup}.json`);

      fs.promises.writeFile(fileName, JSON.stringify(tokenProps));
    }
  });
}
```

Reply:
Is fs.promises a native node module or are you referring to this one https://www.npmjs.com/package/fs.promises ?

Reply:
It's native, you can access the promisified `fs` APIs from `fs.promises.<api>` or `const fsPromises = require('fs/promises)`. I usually just use `fs.promises...` to reduce the number of imports since not all sync `fs` APIs are promisified e.g. `fs.existSync`